package.json

```diff
@@ -1,7 +1,10 @@
 {
   "name": "gridicons",
   "version": "2.1.3",
-  "main": "build/index.js",
+  "main": "cjs/index.js",
+	"files": [
+		"cjs"
+	],
```

Comment:
Indentation 😁
I'd include a slash (`cjs/`) to make clear it's a directory (not just a file).
Can we also include `esm/`?